frontend/src/app/main/ui/shapes/text/fontfaces.cljs

```diff
@@ -76,14 +76,10 @@
   {::mf/wrap-props false
    ::mf/wrap [#(mf/memo' % (mf/check-props ["shapes"]))]}
   [props]
-  (let [shapes  (->> (obj/get props "shapes")
-                     (filterv #(= :text (:type %))))
-
-        content (->> shapes (mapv :content))
-
-        ;; Retrieve the fonts ids used by the text shapes
-        fonts (->> content
-                   (mapv fonts/get-content-fonts)
+  (let [;; Retrieve the fonts ids used by the text shapes
+        fonts (->> (obj/get props "shapes")
+                   (filterv #(= :text (:type %)))
```

Comment:
`cph/test-shape?` predicate should be used here (i know that is not new, but I think we should change it gradually to named predicates that facilitates the readability)

Reply:
ok